src/ast/irbuilderbpf.cpp

```diff
@@ -1474,6 +1474,16 @@ Value *IRBuilderBPF::CreatKFuncArg(Value *ctx,
   return expr;
 }
 
+Value *IRBuilderBPF::CreatRawTracepointArg(Value *ctx, const std::string &builtin)
+{
+  int offset;
+
+  // argX
+  offset = atoi(builtin.substr(3).c_str());
+
+  return CreateRegisterRead(ctx, offset, builtin);
```

Comment:
TBH, I don't like that this calls `CreateRegisterRead` when there are in fact no registers. In addition, `CreateRegisterRead` assumes read from `pt_regs` whose elements have the size of a pointer while raw tracepoint args are always u64. We should probably factor out a piece of `CreateRegisterRead` and use that from here.

Reply:
Yes, it has been re-implemented.